doc/classes/EditorResourcePicker.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="EditorResourcePicker" inherits="HBoxContainer" version="4.0">
+	<brief_description>
+		Godot editor's control for selecting [Resource] type properties.
+	</brief_description>
+	<description>
+		This is a [Control] node similar to the one used in the Inspector dock when editing [Resource]s. It provides options for creating, loading, saving and converting resources.
+
+		[b]Note:[/b] It does not include an editor for the resource.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="can_drop_data_fw" qualifiers="const">
+			<return type="bool">
+			</return>
+			<argument index="0" name="point" type="Vector2">
+			</argument>
+			<argument index="1" name="data" type="Variant">
+			</argument>
+			<argument index="2" name="from" type="Control">
+			</argument>
+			<description>
+			</description>
+		</method>
+		<method name="drop_data_fw">
+			<return type="void">
+			</return>
+			<argument index="0" name="point" type="Vector2">
+			</argument>
+			<argument index="1" name="data" type="Variant">
+			</argument>
+			<argument index="2" name="from" type="Control">
+			</argument>
+			<description>
+			</description>
+		</method>
+		<method name="get_drag_data_fw">
+			<return type="Variant">
+			</return>
+			<argument index="0" name="point" type="Vector2">
+			</argument>
+			<argument index="1" name="from" type="Control">
+			</argument>
+			<description>
+			</description>
+		</method>
+	</methods>
```

Comment:
These methods need description too. Or you can bind them using name starting with `_` so they are hidden.

Reply:
Do we just copy and paste description from a parent class? I've checked some other class when I was doing one of these PRs and there everything was left empty because these are virtual methods and they are described in one of s parent classes.

Reply:
On a second thought, these methods shouldn't even be exposed. I think there is no way to unexpose them other than changing name to something starting with `_`, in which case they wouldn't be called 🤔 So maybe it's fine.

Reply:
If they're from a parent class they shouldn't be added to the XML template at all.

But here they don't exist in the parent class, they are *called* if they exist by `Control`, but they're not C++ virtual methods exposed to the scripting interface.

Chances are that those don't need to be exposed either, just implemented in the class?

Reply:
Right, looks like they don't need to be bound. They are virtual methods.

Reply:
I'm confused by your discussion. Are you saying they don't need `ClassDB::bind_method` to function? Because that's how it's done in [other places](https://user-images.githubusercontent.com/11782833/116546848-cd06d700-a8fa-11eb-9b53-d236f8378a2b.png), including the place where I extracted this code from. And they do seem to generate in docs for [other public facing classes](https://user-images.githubusercontent.com/11782833/116547277-5ae2c200-a8fb-11eb-8638-b4fdb3125d67.png).

Though I do admit that my memory betrayed me and there doesn't seem to be a doc that has descriptions for them, I may have been thinking about something else. I assumed they were described in the `Control.xml`, but alas.

Reply:
Check LineEdit. It uses drag/drop methods without binding them.

Reply:
Hmm, `LineEdit` doesn't use forwarding though. These methods are for forwarding drag and drop from the button control to the parent control here. Now, how important it is to have forwarding here I'm not sure, but it is tied to `EditorNode::drag_resource` and you cannot drop anything on arrow to the right by design.

Reply:
So maybe let's just ignore it for now. Addressing https://github.com/godotengine/godot/issues/16863#issuecomment-829151558 will resolve this.